drivers/SmartThings/sonos/src/sonos_driver.lua

```diff
@@ -184,11 +184,15 @@ function SonosDriver:notify_augmented_data_changed(update_kind, update_key, upda
     )
   end
 
-  if
-    self.oauth.endpoint_app_info
+  local maybe_token, _ = self:get_oauth_token()
```

Comment:
Doesn't need to be in this PR, but I was initially a bit confused, and I think this funciton would be more descriptive if it was `get_cached_oauth_token` as at first read I assumed this was making to request.

Reply:
:ack: